app/policies/project_test_policy.rb

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2015 ProbeDock
+# Copyright (c) 2012-2014 Lotaris SA
+#
+# This file is part of ProbeDock.
+#
+# ProbeDock is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ProbeDock is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ProbeDock.  If not, see <http://www.gnu.org/licenses/>.
+class ProjectTestPolicy < ApplicationPolicy
+  def show?
+    user.try(:is?, :admin) || record.project.organization.try(:public?) || user.try('member_of?', record.project.try(:organization))
```

Comment:
Please use `admin?` instead of `user.try(:is?, :admin)`. Common checks are defined in [this helper class](https://github.com/probedock/probedock/blob/4d6281556b3efb7845647c1905d72830896ef9c9/app/policies/concerns/policy_helpers.rb#L50-L53).

While you're at it, you could add a `member_of?` helper to the class that takes the organization as argument. We'll replace assertions in other policy classes as we go forward.

Please use a symbol, not a string: `:member_of?` instead of `'member_of?'`.

Your last `try` call is either late or superfluous. If the organization was nil, the second `try` call in `record.project.organization.try(:public?)` would fail before reaching it. In the case of a show action for the project, the organization is guaranteed to be here anyway, so you could remove both of these `try` calls.